.changeset/spotty-rats-work.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-react': patch
```

Comment:
Was this supposed to be for `plugin-search`? No need for a changeset though, since it's just changing the stories

Reply:
wops yes! and yes good point!